src/app/data-planes/views/DataPlaneListView.vue

```diff
@@ -143,4 +143,8 @@ import type { MeSource } from '@/app/me/sources'
   // Removes the bottom margin as we’re aligning the label with the select in a horizontal layout.
   margin-bottom: 0 !important;
 }
+
+.data-plane-collection :deep(.name-column) {
+  max-width: 400px;
+}
```

Comment:
How come we split this CSS in two? As it is now would mean that the Service > Data Plane Proxy listing wouldn't truncate, is that right?

Reply:
@kleinfreund wasn't sure if you'd seen this?

Reply:
I had, but forgot about it again. Thanks for the ping.

I’ll have a look at this now. I wasn’t entirely sure about the approach initially so I only rolled it out in one place. I’ll make sure it also applies to the other DPP listing.

Reply:
I changed this to apply to both Dataplane lists now.